Original patch from: gentoo/src/patchsets/glibc/2.9/6018_all_alpha-glibc-2.8-cache-shape.patch

-= BEGIN original header =-
older verisons of glibc would build dl-sysdep as shared-only and dl-support as
static-only.  alpha hooks in a cache variable via dl-auxv.h.  newer versions of
glibc build dl-sysdep as both shared and static which means we now have symbol
duplication for static builds with dl-sysdep and dl-support.  since dl-sysdep
is both shared/static, there is no point in hooking dl-support anymore, so we
can punt it.

-= END original header =-

diff -durN glibc-2_9.orig/sysdeps/unix/sysv/linux/alpha/dl-support.c glibc-2_9/sysdeps/unix/sysv/linux/alpha/dl-support.c
--- glibc-2_9.orig/sysdeps/unix/sysv/linux/alpha/dl-support.c	2007-03-13 22:25:16.000000000 +0100
+++ glibc-2_9/sysdeps/unix/sysv/linux/alpha/dl-support.c	2009-02-02 22:01:37.000000000 +0100
@@ -1,2 +0,0 @@
-#include "dl-auxv.h"
-#include <elf/dl-support.c>
